website/.eleventy.cjs

```diff
@@ -29,8 +30,26 @@ module.exports = function (eleventyConfig) {
     })
   );
 
+  /**
+   * Generate a relative path to the root from the given page URL.
+   *
+   * Useful when a template which is used from different directories needs to
+   * reliably refer to a path with a relative URL, so that the site can be
+   * served from different sub-directories.
```

Comment:
How are you serving the site locally? I've handled this before by keeping `pathPrefix` and using the `--base-path` flag for Web Dev Server.

Reply:
Everything now uses a relative path, so it works from any directory as long as the server redirects to append `/`

Reply:
More context, we want this so that we can serve PR previews using the technique discussed here: https://github.com/google/wireit/pull/200#issuecomment-1121688775